app/models/movie_person_credits.rb

```diff
@@ -1,30 +1,25 @@
-class MoviePersonCredits
+# frozen_string_literal: true
 
-  def initialize(movies, tv, directing, editing, writing, screenplay, producer, actor)
```

Comment:
Strangely enough, I don't think `movies` and `tv` were doing anything here. If you follow the code down the pipe, you'll see that the data for `movies` and `tv`is getting populated later. Perhaps the instance variables were created here in order to populate them later? Dunno. But they're gone now and stuff still works. 🧹